lib/src/widgets/buttons/ds_request_location_button.widget.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSRequestLocationButton extends DSButton {
+  DSRequestLocationButton({
+    super.key,
+    super.label,
+  }) : super(
+          onPressed: null,
```

Comment:
We could have a function to pass on `onPressed`, similar to `DSQuickReply` widget, this is necessary when the client uses blip chat.

What do you think?